ios/ble/central/Central.swift

```diff
@@ -70,4 +68,14 @@ class Central: NSObject, CBCentralManagerDelegate {
             connectedPeripheral.writeValue(messageData, for: characteristic, type: .withoutResponse)
         }
     }
+    
+    func disconnectAndClose() {
+        if let connectedPeripheral = self.connectedPeripheral {
+            centralManager.cancelPeripheralConnection(connectedPeripheral)
+        }
+    }
+
+    func destroyConnection(){
```

Comment:
1. Move this method into Wallet.swift file. The method scope should leak into Central layer. If required place a delegate method as part of Wallet/PeripheralCommunicatorDelegate. 
2. Also rename this function to `onDeviceDisconnected` for consistency

Reply:
done with suggested changes.